examples/posenet/utils.js

```diff
@@ -133,17 +128,15 @@ class Utils{
     this.offsetTensor = new Float32Array(this.OFFSET_TENSOR_SIZE);
     this.displacementFwd = new Float32Array(this.DISPLACEMENT_FWD_SIZE);
     this.displacementBwd = new Float32Array(this.DISPLACEMENT_BWD_SIZE);
-    this.model = new PoseNet(this.modelArch, Number(this._version), this._use_atrous_conv, Number(this._outputStride),
+    this.model = new PoseNet(this.modelArch, Number(this._version), this._useAtrousConv, Number(this._outputStride),
                              this.scaleInputSize, this._type, this._cacheMap, backend, prefer);
     let start = performance.now();
     result = await this.model.createCompiledModel();
     console.log(`compilation result: ${result}`);
     let elapsed = performance.now() - start;
     console.log(`Compilation time: ${elapsed.toFixed(2)} ms`);
```

Comment:
The compilation time will be output in the `this.model.createCompiledModel();`. The name of `createCompiledModel()` is quite confusing. And now there is no way to get the compilation time outside.

Reply:
Removed compilationTime code